File: src/lugbygame/Ball.java
package lugbygame;

public class Ball extends Unit {

	public Ball(int x, int y) {
		super(x, y);
	}

	public void fly2(int x, int y) {
		this.x = x;
		this.y = y;
	}

	public void dribbled(Player p) {
		this.x = p.x;
		this.y = p.y;
	}

	public int getX() {
		return x;
	}

	public int getY() {
		return y;
	}
	
	public String toString() {
		return "$";
	}
}

File: lucby_edit/Ball.py
from Unit import Unit

class Ball(Unit):
	
	def __init__(self,x,y):
		super().__init__(x=x,y=y)
	
	def fly2(self, x,y):
		self.x=x
		self.y=y
	
	def dribbled(self, p):
		self.x=p.x
		self.y=p.y
	
	def getX(self):
		return int(self.x)
	def getY(self):
		return int(self.y)
	
	def __str__(self):
		return "$"